package.json

```diff
@@ -97,16 +95,16 @@
 		"yup": "^1.4.0"
 	},
 	"devDependencies": {
-		"@cucumber/cucumber": "^9.6.0",
+		"@cucumber/cucumber": "^10.8.0",
 		"@ledgerhq/hw-transport-mocker": "^6.29.0",
 		"@testing-library/dom": "^8.20.1",
 		"@testing-library/jest-dom": "^5.17.0",
 		"@testing-library/react": "^13.4.0",
 		"@testing-library/react-hooks": "^8.0.1",
 		"@testing-library/user-event": "13.5.0",
+		"@types/dlv": "^1.1.4",
 		"@types/postcss-import": "^14.0.3",
 		"@types/react": "^18.3.3",
-		"@types/react-datepicker": "^4.19.6",
```

Comment:
https://github.com/Hacker0x01/react-datepicker/releases/tag/v7.0.0
not needed anymore